### DISCUSSION TITLE
Predict using a model trained on a custom dataset

### DISCUSSION THREAD

**Discussion by anuraganand1838** [open-mmlab/mmaction2#814] [Q&A]
Using this[ model](https://github.com/open-mmlab/mmaction2/blob/master/demo/mmaction2_tutorial.ipynb) I have trained on a custom dataset. Now I want to know the classification results(i.e. model prediction results) on my validation set. Please tell how to do this.

**Comment by dreamerlin** [comment]
Just finetune and test on your dataset. For more details, please refer to the documentation!
